client/components/site-stats-sticky-link/index.jsx

```diff
@@ -12,6 +13,8 @@ var SiteStatsStickyLink = React.createClass( {
 		onClick: React.PropTypes.func
 	},
 
+	_preloaded: false,
```

Comment:
This feels dirty with regards to React. Why not just store this in the component `state` or add it into the logic for `sections`?